packages/http-client-csharp/package.json

```diff
@@ -34,12 +34,16 @@
     "test:ci": "vitest run --coverage --reporter=junit --reporter=default",
     "lint": "eslint . --max-warnings=0",
     "lint:fix": "eslint . --fix",
-    "format": "pnpm -w format:dir packages/http-client-csharp"
+    "format": "pnpm -w format:dir packages/http-client-csharp",
+    "extract-api": "npx api-extractor run --local --verbose"
   },
   "files": [
     "dist/**"
   ],
   "dependencies": {
+    "@azure-tools/ts-genapi": "^1.0.7",
```

Comment:
where is this package coming from its not on npm

Reply:
it comes from running the extract command locally

Reply:
I think the package itself is from our internal feed

Reply:
Ok, confirmed those only get added when you run Generate-APIView-CodeFile.ps1.  This script needs them to convert the *.api.json file into a file apiview can consume *_js.json.

Reply:
yeah it be great if that work with them not included in your last change as this basically now makes it impossible to contribute for an external user.